### PR TITLE
Release version 7.17.0.0

### PR DESCRIPTION
**Pull request by benj-moreau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,11 +15,31 @@ This is a multi bucket aggregation.
 Installation
 ------------
 
-`bin/plugin --install path_hierarchy --url "https://github.com/opendatasoft/elasticsearch-aggregation-pathhierarchy/releases/download/v7.6.0.0/pathhierarchy-aggregation-7.6.0.0.zip"`
+`bin/plugin --install path_hierarchy --url "https://github.com/opendatasoft/elasticsearch-aggregation-pathhierarchy/releases/download/v7.17.0.0/pathhierarchy-aggregation-7.17.0.0.zip"`
 
 Build
 -----
-Requires Java 12
+Requires Java 17
```
Comment (5k4nd) [inline README.md]: Really? Java 15 also does the job, so maybe 15 through 17 is ok.

File: src/main/java/org/opendatasoft/elasticsearch/search/aggregations/bucket/InternalPathHierarchy.java [hunk 2]
```diff
@@ -30,7 +29,7 @@
  * Mainly, returns the builder and makes the reduce of buckets.
  */
 public class InternalPathHierarchy extends InternalMultiBucketAggregation<InternalPathHierarchy,
-        InternalPathHierarchy.InternalBucket> implements PathHierarchy {
+        InternalPathHierarchy.InternalBucket> {
     protected static final ParseField SUM_OF_OTHER_HIERARCHY_NODES = new ParseField("sum_other_hierarchy_nodes");
```
Comment (5k4nd) [inline src/main/java/org/opendatasoft/elasticsearch/search/aggregations/bucket/InternalPathHierarchy.java:33]: Not in use anymore?

**Review by 5k4nd — APPROVED** [review]
From a static reading, everything looks fine.

Also, there is at least one multi-shard test (in 30_date_hierarchy.yml) 👌

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+services:
+  elasticsearch-plugin-debug:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+      target: elasticsearch-plugin-debug
+    environment:
+      - discovery.type=single-node
+      # NO DEBUG
```
Comment (mhugo) [inline docker-compose.yml:11]: DEBUG and NO DEBUG are the same here

Comment (benj-moreau) [reply docker-compose.yml:11]: In contrast to NO DEBUG ES_JAVA_OPTS, DEBUG adds the `-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:5005` in order to attach a debugger to the remote JVM.

Comment (mhugo) [reply docker-compose.yml:11]: Arf, sorry, bad newline formatting, I misread :)

**Review by mhugo — APPROVED** [review]

**Pull request merged**